### DISCUSSION TITLE
Addon: Wie öffne ich ein Popup im Menu

### DISCUSSION THREAD

**Discussion by skerbis** [FriendsOfREDAXO/.github#1] [Q&A] [answered]
Was muss ich in einer `package.yml` eingeben damit der Menüpunkt ein Popup wird?

` popup: true`

reicht nicht. Das legt nur das Design fest.

**Comment by dergel** [comment, marked as answer]
Hi Thomas,

es gibt keinen zentralen Weg, ein Popup zu erzeugen.
Allerdings kannst du statt  `popup: true`  auch ein Script übergeben, Beispiel aus dem Medienpool `popup: openMediaPool(); return false;`, das Script kann man z.B. mit dem project-AddOn in das Backend laden.
`popup: true` stellt nur sicher, dass das UI reduziert wird.
Alternativ kannst du auch ein Popup über ein zusätzliches Attribut ansteuern, sodass du für alle deine Popups dasselbe Script nutzen kannst.

`linkAttr: { data-popup: true }`

**Reply by skerbis** [reply]
Danke @dergel
